ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# Description of problem
+
+(remove this text and list the steps you followed)
```

Comment:
I'd probably lean towards 'replace this text with...', like we have for the `cc-collect-data.sh` section.

Reply:
Agreed. Let me update that...

Reply:
Updated.